support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -17,23 +17,6 @@ export const pageUrlRegexes = {
 		genericCheckoutOnly: '(uk|us|au|ca|eu|nz|int)/checkout|thank-you(/.*)?$',
 	},
 	subscriptions: {
-		subsDigiSubPages: '(/??/subscribe(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)',
```

Comment:
These all are irrelevant now because the digisub page is at `/subscribe/digitaledition`